cmd/kubeadm/app/constants/constants.go

```diff
@@ -538,6 +540,42 @@ func GetDNSIP(svcSubnet string) (net.IP, error) {
 	return dnsIP, nil
 }
 
+// GetKubernetesServiceCIDR returns the default Service CIDR for the Kubernetes internal service
+func GetKubernetesServiceCIDR(svcSubnetList string, isDualStack bool) (*net.IPNet, error) {
+	if isDualStack {
+		// The default address family for your cluster is the address family of the first
+		// service cluster IP range configured via the `--service-cluster-ip-range` flag
+		// to the kube-controller-manager.
```

Comment:
`to the` -> `of the`.
this seems to be a flag to the api-server too.

my understand is that they should match?
should we change this to the following?
`of the kube-controller-manager and kube-apiserver.`

Reply:
Yes, you're right.